pkgs/by-name/um/umu-launcher-unwrapped/package.nix

```diff
@@ -54,6 +61,29 @@ python3Packages.buildPythonPackage rec {
     "SHELL_INTERPRETER=${lib.getExe bash}"
   ];
 
+  # The `umu-vendored` target is for building "vendored" or "bundled" dependencies.
+  # However, this fails and we get them from nixpkgs anyway.
+  #
+  # As a temporary solution, explicitly specify the supported build targets.
+  #
+  # Ideally, upstream will provide a sane way to disable building the vendored
+  # dependencies in the future.
+  buildFlags = [
+    "umu-dist"
+    "umu-launcher"
+  ];
+
+  # Same issue for install targets
+  installTargets = [
+    "umu-dist"
+    "umu-docs"
+    "umu-launcher"
+    "umu-delta"
+    "umu-install"
+    "umu-launcher-install"
+    "umu-delta-install"
```

Comment:
Shouldn't the delta targets only be installed when we have the dependencies for them (withDeltaUpdates)?

Reply:
Upstream say these should always be targeted, regardless of `withDeltaUpdates`